backend/util/helpers.js

```diff
@@ -0,0 +1,206 @@
+const { faker } = require('@faker-js/faker');
+const { Product } = require('../models/product');
+
+async function addFakeData (req, res, next) {
+  for (let i = 0; i < 100; i++) {
+    let product = new Product();
+    const numReviews = Math.floor(Math.random() * 8);
+
+    product.category = faker.commerce.department();
+    product.name = faker.commerce.productName();
+    product.price = faker.commerce.price();
+    product.image = 'https://via.placeholder.com/250?text=Product+Image';
+    
+    // Create fake reviews
+    for (let j = 0; j < numReviews; j++) {
+      const username = faker.internet.userName();
+      const text = faker.lorem.sentence();
+      
+      product.reviews.push(
+        {
+          'username': username,
+          'text': text,
+        }
+      );
+    }
+
+    await product.save()
+      .catch(err => console.error(err));
+  }
+  res.status(200).send('Products saved');
+}
+
+async function getProducts(req, res, next) {
+  let page = Number(req.query.page); 
```

Comment:
```suggestion
  let page = Number(req.query.page) ?? 1
```
Should work the same as the check you are doing in line 38.